README.md

```diff
@@ -16,6 +16,9 @@ admins:
       - <ssh key to deploy>
       ...
     passwd: <hashed passphrase> #optional set password for user, passphrase needs to be hashed (for more information see: http://docs.ansible.com/ansible/faq.html#how-do-i-generate-crypted-passwords-for-the-user-module)
+locales: <array of locales to generate>
+keyboard_layout: <Keyboard layout to select for the TTY>
```

Comment:
Please mention the defaults, or whether these vars are mandatory. Also, I suggest an indented markdown table for role variables.